frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Is this needed, since there's a .gitignore at the top level?

Reply:
I was following the structure of the mern app... we didn't remove it from that so I didn't really think about it. I can see why it wouldn't be necessary though, shall I delete it?